internal/builders/go/pkg/build_test.go

```diff
@@ -1026,6 +1026,7 @@ func Test_generateCommand(t *testing.T) {
 			}
 
 			for _, cfg := range cfgs {
+				cfg := cfg
```

Comment:
I think that if the slice definition (at line 1016) is for a pointer then we can avoid this issue.

i.e.
```
cfgs := []*goReleaserConfigFile{
    ...
}
```

then line 1029 can just be

```golang
c, err := fromConfig(cfg)
```

Reply:
Yes, good point!